src/Sylius/Bundle/PayumBundle/Resources/config/services/validator.xml

```diff
@@ -11,20 +11,23 @@
 
 -->
 
-<container xmlns="http://symfony.com/schema/dic/services"
-           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
+<container
+    xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
 >
     <services>
         <defaults public="true" />
 
         <service id="Sylius\Bundle\PayumBundle\Validator\GatewayFactoryExistsValidator">
             <argument>%sylius.gateway_factories%</argument>
             <tag name="validator.constraint_validator" alias="sylius_gateway_factory_exists_validator" />
+            <deprecated package="sylius/payum-bundle" version="1.14">The "%service_id%" service is deprecated since 1.14 and will be moved to the PayumBundle in 2.0.</deprecated>
```

Comment:
```suggestion
            <deprecated package="sylius/payum-bundle" version="1.14">The "%service_id%" service is deprecated since 1.14 and will be moved to the PaymentBundle in 2.0.</deprecated>
```